package.json

```diff
@@ -22,9 +22,11 @@
   ],
   "dependencies": {
     "csv-parse": "^4.8.8",
-    "discord.js": "^11.6.3",
-    "luxon": "^1.22.0",
-    "node-fetch": "^2.6.0"
+    "discord.js": "^11.6.4",
+    "eslint": "^6.8.0",
+    "luxon": "^1.23.0",
+    "node-fetch": "^2.6.0",
+    "tape": "^4.13.2"
   },
```

Comment:
Should remain a devDependency because if we ever publish as a package, package consumers don't need it